mod/wet4/views/default/page/components/module.php

```diff
@@ -143,6 +143,16 @@
         $header = elgg_format_element('h2', ['class' => 'panel-title'], $title);
     }
 
+    // CL 23082022 - Coll
+    $owner = elgg_get_page_owner_entity()->username;
+    if ($title === 'Colleagues') {
+        $header = elgg_format_element('h2', ['class' => 'panel-title'], "$owner's $title");
+    }
+    if ($title === 'collègues') {
+        $header = elgg_format_element('h2', ['class' => 'panel-title'], "$title à $owner");
+    }
```

Comment:
I tested this and it seems like when you change the page to French it doesn't pickup the profile name, it just says "Collègues". I think it may be overwritten somewhere else since "Collègues" is capitalized but the FR string you edited is lowercase "collègues". You might also want to switch the second if statement to an else if.

Reply:
That's strange since it's changing for me in my dev env. I will however, fix the capitalization for both the Title and the page owner. I know the caching is a bit of a pain with Elgg and I've had to clear cache and refresh a couple times before my changes were reflected
![image](https://user-images.githubusercontent.com/17506947/186230856-2a9859a7-5fce-4552-a97e-0224461d7cf3.png)

Reply:
Hmm, I tried flushing the cache and I get the same result. I put a console log in the code and it does pick up my name and title so I have no clue why it's only showing Collègues for the French version. @piet0024 can you take a look? 

![image](https://user-images.githubusercontent.com/2327968/186235104-e0a86eaa-de8d-4fb2-9514-f0e56faec9d5.png)

Reply:
I pulled your new changes and can see now that the English version has a capitalized first name meaning my code is up to date. I still have no name for the French version. Not sure what's going on, but since it's just on my side I will resolve this. Might want to keep an eye out to see how it behaves in prod though.